spec/lib/adviser_csv_lookup_spec.rb

```diff
@@ -0,0 +1,62 @@
+RSpec.describe AdviserCsvLookup do
+  let(:adviser1) { FactoryGirl.create(:adviser, adviser1_certifications) }
+  let(:adviser2) { FactoryGirl.create(:adviser, adviser2_certifications) }
+  let(:advisers) { [adviser1, adviser2] }
+
+  let(:qualifications) { Array.new(3) { FactoryGirl.create(:qualification) } }
+  let(:accreditations) { Array.new(3) { FactoryGirl.create(:accreditation) } }
+
+  let(:adviser1_certifications) do
+    {
+      qualifications: [qualifications.first, qualifications.last],
+      accreditations: [accreditations.first]
+    }
+  end
+  let(:adviser2_certifications) do
+    {
+      qualifications: [qualifications.last],
+      accreditations: [accreditations.last]
+    }
+  end
+
+  let(:subject) { described_class.new(advisers) }
+
+  describe '#each' do
+    let(:perform) { subject.each {} }
```

Comment:
I'm not sure `perform` is the best name for this variable.

Reply:
Hi @margOnline, happy to change this, but not sure what a better alternative would be here. Do you have any ideas?